scripts/db_backup.sh

```diff
@@ -134,45 +143,49 @@ use_env_values()
 
 add_temp_mysql_user()
 {
-    #todo: conditionalize based on whether temp credentials have been provided
-    log "Adding ${TEMP_DATABASE_USER} to db"
-    touch $TMP_QUERY_ADD
-    chmod 700 $TMP_QUERY_ADD
+    if [ ! -z $TEMP_DATABASE_USER ] && [ ! -z $TEMP_DATABASE_PASSWORD ]; then
```

Comment:
our invocations of this file always provide them, but they're technically "optional" params. i think it's good practice to check values before using them.